api/src/main/java/edu/cornell/mannlib/vitro/webapp/visualization/temporalgraph/TemporalGrantVisualizationRequestHandler.java

```diff
@@ -42,12 +42,12 @@
 
 public class TemporalGrantVisualizationRequestHandler implements
 		VisualizationRequestHandler {
-	

```

Comment:
Unnecessary formatting changing make it challenging to determine if there are functional updates buried in the noise. Please revert these updates.